healthyregions/SDOHPlace
# fix: Address all feedback from Design Audit 11/11/2024

## Problem
We've noticed some styling issues with the current SDOH website

This feedback has been recorded in the [Design Audit 11/11/2024 notes in Notion](https://www.notion.so/UX-Audit-2024-25-2a26f4fcfc394fe28150b5f26caacddc)

## Approach
* fix: Navbar (Desktop) - added SDOH Logo to desktop navbar in all views except for Home (logo is already displayed there)
* fix: Navbar (Mobile) - left-align menu options here, add Logo to all views
* feat: Navbar (Mobile) - added animation to the Menu/Close icon when it transitions from one to the other
* fix: Navbar dotted lines - fixed aspect ratio for `line1`, adjusted width of `line4` to match the others
* fix: Navbar - added additional space between each nav button, move navbar down slightly, and moved all views down slightly to accommodate this change
* fix: Navbar - added some extra padding between dropdown button and the menus that open when they are clicked
* feat: Navbar - added `box-shadow` to dropdown menus
* fix: Homepage - add some padding beneath the left side large logo to better align it with the right side
* fix: Homepage (Mobile) - adjust margins/padding to better balance the whitespace in this view, decrease total height by `1%` so that first panel fits on the screen (iPhone 12)

## How to Test
1. Navigate to homepage: https://deploy-preview-377--cheerful-treacle-913a24.netlify.app/
    * You should see the updated styles here for the Desktop Navbar and Desktop Homepage
2. Switch to mobile view
    * You should see the updated styles here for the Mobile Navbar and Mobile Homepage
3. Check other views to make sure nothing that drastic has changed
    * You should see that all views have had their content moved down slightly to accommodate the Navbar changes
    * You should see that SDOH Logo now appears in the Desktop Navbar on all views except for the Homepage

### Screenshots (Desktop)
![Screenshot 2024-11-13 at 12 51 56 PM](https://github.com/user-attachments/assets/250d8472-c850-4f50-a731-44dfcebf1ab8)

![Screenshot 2024-11-13 at 12 52 11 PM](https://github.com/user-attachments/assets/feaca3c5-ab3b-4ea6-b0f0-8d452e045318)


### Screenshots (Mobile)
![Screenshot 2024-11-13 at 12 51 39 PM](https://github.com/user-attachments/assets/36a3d49e-adb8-4a03-a3e6-7a0e5830d3e7)

![Screenshot 2024-11-13 at 12 51 45 PM](https://github.com/user-attachments/assets/6da7f7ad-1788-4aa8-b8eb-94999882b758)
